code/modules/clothing/head/misc_special.dm

```diff
@@ -218,3 +218,22 @@
 	hair_style = pick(GLOB.hair_styles_list - "Bald") //Don't want invisible wig
 	hair_color = "#[random_short_color()]"
 	. = ..()
+
+/obj/item/clothing/head/foilhat
+	name = "tinfoil hat"
+	desc = "Thought control rays, psychotronic scanning. Don't mind that, I'm protected cause I made this hat."
+	icon_state = "foilhat"
+	item_state = "foilhat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = -5,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = -5, "fire" = 0, "acid" = 0)
+
+/obj/item/clothing/head/foilhat/equipped(mob/living/carbon/human/user)
+	user.gain_trauma(/datum/brain_trauma/mild/phobia, FALSE, "security", "lizards")

```

Comment:
This actually doesn't work, the "lizards" argument is not used so this will only be security, you may want to modify the New proc of phobia to also support the `specific_type` argument being a list of phobias

Reply:
didn't notice that, can do